odoo/models.py

```diff
@@ -63,6 +64,7 @@
 _logger = logging.getLogger(__name__)
 _schema = logging.getLogger(__name__ + '.schema')
 _unlink = logging.getLogger(__name__ + '.unlink')
+_logger_clear_caches = logging.getLogger(__name__ + '.clear_caches')
```

Comment:
FYI `logging` has a method to get child loggers:

```suggestion
_logger_clear_caches = _logger.getChild('clear_caches')
```

Reply:
Good to know!

Thank you!

I just copy&paste the previous line of odoo

What about creating a new PR to odoo to change all these children?

This PR is just a temporal dummy